### DISCUSSION TITLE
Wiki based on tiptap?

### DISCUSSION THREAD

**Discussion by guettli** [ueberdosis/tiptap#2222] [General Questions] [answered]
Is there already a wiki based on tiptap?

For my usecase editing markdown by hand is not a solution.

I need a WYSIWYG solution.

**Comment by philippkuehn** [comment, marked as answer]
GitLab is using tiptap for it’s new wiki editor.

**Reply by guettli** [reply]
Unfortunately the Gitlab Wiki is missing a lot of features. Like back-links, global history, global search.

But maybe Gitlab is improving this at the moment. Would be great.

**Reply by philippkuehn** [reply]
Yeah it’s still in beta. [Notebag](https://notebag.app/) is also a great app based on tiptap.

**Reply by guettli** [reply]
> Yeah it’s still in beta. [Notebag](https://notebag.app/) is also a great app based on tiptap.

Thank you. Notebag looks great for personal notes. I am looking for a web-based tool which can be used by a team.
